package.json

```diff
@@ -28,21 +28,20 @@
   "scripts": {
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
-    "postinstall": "scripts/postinstall",
+    "prepare": "scripts/prepare",
     "test": "scripts/test",
     "testOnly": "scripts/testOnly",
-    "prepare": "./scripts/prepare",
     "build": "./scripts/build",
     "prettier:fix": "npx prettier --write --ignore-unknown .",
     "prettier": "npx prettier --check ."
   },
   "dependencies": {
     "@openeth/truffle-typings": "0.0.6",
     "@openzeppelin/contracts": "^3.2.0",
-    "@rsksmart/rif-relay-client": "https://github.com/infuy/rif-relay-client",
-    "@rsksmart/rif-relay-common": "https://github.com/infuy/rif-relay-common",
-    "@rsksmart/rif-relay-contracts": "https://github.com/anarancio/rif-relay-contracts",
-    "@rsksmart/rif-relay-server": "https://github.com/infuy/rif-relay-server",
+    "@rsksmart/rif-relay-client": "0.0.2-alpha.3",
+    "@rsksmart/rif-relay-common": "0.0.2-alpha.4",
+    "@rsksmart/rif-relay-contracts": "0.0.2-alpha.6",
+    "@rsksmart/rif-relay-server": "rsksmart/rif-relay-server#PP-271/fix-integration-tests",
```

Comment:
This line needs to be updated after the [server PR](https://github.com/rsksmart/rif-relay-server/pull/27) is merged